homeassistant/components/panasonic_viera/const.py

```diff
@@ -18,4 +18,7 @@
 ATTR_MODEL_NUMBER = "modelNumber"
 ATTR_UDN = "UDN"
 
+DEFAULT_MANUFACTURER = "Panasonic"
+DEFAULT_MODEL_NUMBER = "Panasonic Viera"
```

Comment:
Shouldn't it be just "Viera" ?

Reply:
that's a matter of discussion, haha
but yes, now that I think about it, Viera would be better